recipes/scalene/meta.yaml

```
@@ -0,0 +1,58 @@
+{% set name = "scalene" %}
+{% set version = "1.5.10" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  # See https://pypi.org/project/scalene/#files
+  sha256: 8e2e2ae14acdca56d5bbf33e158814702410f0a72d4e690fce837b811ffadbfc
+
+build:
+  skip: true  # [py<38]
+  skip: true  # [win]
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - make
+  host:
+    - python
+    - pip
+    - setuptools_scm
+    - wheel
+  run:
+    - python
+    - rich >=9.2.0
+    - cloudpickle >=1.5.0
+    - numpy
+    - pynvml >=11.0.0
+
+test:
+  imports:
+    - {{ name }}
+  source_files:
+    - test/original/bm_mdp.py
+  requires:
+    - pyperf
+    - pip
+  commands:
+    - pip check
+    - {{ PYTHON }} -m scalene --cli test/original/bm_mdp.py  # [not win]
+    - {{ PYTHON }} -m scalene --cli test\\original\\bm_mdp.py  # [win]
```

Since `{{ PYTHON }}` seems to be expand to `python` in the `host` environment, using `{{ PYTHON }}` here doesn't seem to work. Would just use `python`

```suggestion
    - python -m scalene --cli test/original/bm_mdp.py  # [not win]
    - python -m scalene --cli test\\original\\bm_mdp.py  # [win]
```

---

Fingers crossed!

---

If it just expands to "python", won't this have the same effect? I will try it but next thing to try seems to be "python3". (Unless there is a Jinja variable`PYTHON3`...)

---

`{{ PYTHON }}` is Jiinja syntax. This gets filled in by an initial parsing of the recipe. Ideally `{{ PYTHON }}` would be set to something context dependent, but that doesn't not appear to be the case (hence why the path to the `host` `python` is used in the `test` phase).

It shouldn't present an issue as things like `python` are not expanded. The `{{ ... }}` syntax is needed to look up a Jinja variable and it expand it (as happens with `{{ PYTHON }}`)